3scale/porta
# Fix stats

**What this PR does / why we need it**:

> Currently a module from Stats is broken in SaaS. 

**Which issue(s) this PR fixes** 

fixes https://issues.jboss.org/browse/THREESCALE-2957
and fixes https://issues.jboss.org/browse/THREESCALE-2183 (for free)

**Verification steps** 

Browse to a Service and then click on "Analytics --> Usage" on the left menu. Then everything should work as expected, no error in console.
